server/index.js

```diff
@@ -1,25 +1,26 @@
+/* eslint-disable no-console */
+
 require('dotenv').config();
 const path = require('path');
 const express = require('express');
-const { db } = require('./database.js');
+const { Room } = require('./database.js');
 
 const app = express();
 
 app.use(express.static(path.join(__dirname, '..', 'public')));
 app.use(express.json());
 app.use(express.urlencoded());
 
-
 app.route('/api/room/:id')
   .get((req, res) => {
-    // database retrieval goes here
-    // placeholder:
-    res.sendStatus(200);
-  })
-  .post((req, res) => {
-    // database insertion goes here
-    // placeholder:
-    res.sendStatus(201);
+    console.log('ID:', req.params.id);
+    Room.find({ id: req.params.id }, (err, record) => {
+      if (err) {
+        console.log('Retrieval error:', err);
+        res.sendStatus(500);
+      }
+      res.status(200).send(record);
```

Comment:
You may want to consider either including this line in an else statement or inserting a return statement in your error block. (Otherwise, you'll reach this line even if there's an error.)

Reply:
In Express, sending the response ends the middleware processing. On an error, line 20 sends a status 500 error back to the client, so if line 20 runs, it's the last line that runs.